backend/HoReD/Models/SalaryBindingModel.cs

```diff
@@ -1 +1,18 @@
-﻿
\ No newline at end of file
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace HoReD.Models
+{
+    public class SalaryBindingModel
+    {
+        public int professionId { get; set; }
+
+        public double rate { get; set; }
```

Comment:
?? property should be written in upper-case

Reply:
@olyasol will be fixed!